React 16 Support

Currently this is blocked on an upstream dependency, `react-native-web` - they're tracking this at https://github.com/necolas/react-native-web/issues/364

When trying to run against react@16.0.0-beta.5, it looks like:
```
Error: Cannot find module 'react-dom/lib/EventPluginHub'
    at Function.Module._resolveFilename (module.js:485:15)
    at Function.Module._load (module.js:437:25)
    at Module.require (module.js:513:17)
    at require (internal/module.js:11:18)
    at Object.<anonymous> (/Users/-/Code/zendesk/protoloop-4/node_modules/react-native-web/dist/modules/injectResponderEventPlugin/index.js:1:83)
    at Module._compile (module.js:569:30)
    at Object.Module._extensions..js (module.js:580:10)
    at Module.load (module.js:503:32)
    at tryModuleLoad (module.js:466:12)
    at Function.Module._load (module.js:458:3)
```

--- a/package.json
+++ b/package.json
@@ -74,6 +74,7 @@
   "dependencies": {
     "animated": "^0.1.5",
     "asap": "^2.0.5",
+    "create-react-class": "^15.6.2",
     "deline": "^1.0.4",
     "flexibility": "^2.0.1",
     "inline-style-prefixer": "^2.0.5",

--- a/src/modules/Touchable.js
+++ b/src/modules/Touchable.js
@@ -1,4 +1,5 @@
 const React = require('react');
+const createReactClass = require('create-react-class');
 const PropTypes = require('prop-types');
 const TimerMixin = require('react-timer-mixin');
 
@@ -74,7 +75,7 @@ const Touchable = (
   });
 
   // eslint-disable-next-line react/prefer-es6-class
-  return React.createClass({
+  return createReactClass({
     displayName: 'Touchable',
     propTypes: {
       accessible: PropTypes.bool,

--- a/src/vr/Touchable.js
+++ b/src/vr/Touchable.js
@@ -1,4 +1,5 @@
 const React = require('react');
+const createReactClass = require('create-react-class');
 const PropTypes = require('prop-types');
 const { VrButton } = require('react-vr');
 
@@ -54,7 +55,7 @@ const Touchable = (
   TouchableMixin,
 ) => {
   // eslint-disable-next-line react/prefer-es6-class
-  return React.createClass({
+  return createReactClass({
     displayName: 'Touchable',
     propTypes: {
       accessible: PropTypes.bool,

--- a/src/web/Touchable/Touchable.js
+++ b/src/web/Touchable/Touchable.js
@@ -1,4 +1,5 @@
 const React = require('react');
+const createReactClass = require('create-react-class');
 const TimerMixin = require('react-timer-mixin');
 const TouchableMixin = require('./TouchableMixin');
 const ensurePositiveDelayProps = require('./ensurePositiveDelayProps');
@@ -64,7 +65,7 @@ const Touchable = (Animated, StyleSheet, Platform) => {
   });
 
   // eslint-disable-next-line react/prefer-es6-class
-  return React.createClass({
+  return createReactClass({
     displayName: 'Touchable',
     propTypes: {
       accessible: PropTypes.bool,

